include/tlog/timespec.h

```diff
@@ -52,6 +52,11 @@ extern const struct timespec tlog_timespec_max;
 #define TLOG_TIMESPEC_FP_MIN \
             nextafter((double)LONG_MIN - (double)0.999999999, 0)
 
+/** Nanoseconds per second */
+#define TLOG_TIMESPEC_NSEC_PER_SEC  1000000000
+
+/** Negative Nanoseconds per second */
+#define TLOG_TIMESPEC_NEG_NSEC_PER_SEC  -1000000000
```

Comment:
Please don't create another constant. Just put a minus in front of the one above when you need it. It will be more readable.